packaging/unix/MUSIC.sh.in

```diff
@@ -31,7 +31,7 @@ MEDINRIA_BIN="@MEDINRIA_BIN@"
 #   Set the plugins and library paths.
 
 export MEDINRIA_PLUGIN_PATH="@MEDINRIA_PLUGINS_DIRS@"
-export LD_LIBRARY_PATH=${MEDINRIA_DIR}/lib:${MEDINRIA_DIR}/lib/InsightToolkit:${MEDINRIA_DIR}/lib/vtk-5.10:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=${MEDINRIA_DIR}/lib:${MEDINRIA_DIR}/DCMTK/install/lib:$LD_LIBRARY_PATH
```

Comment:
DCMTK libs were not found

Reply:
And the others were not used, as in the [VTK7 PR](https://github.com/Inria-Asclepios/medinria-superproject/pull/28).